stable/coredns/values.yaml

```diff
@@ -67,6 +67,10 @@ servers:
 # - zones:                 # the `zones` block can be left out entirely, defaults to "."
 #   - zone: hello.world.   # optional, defaults to "."
 #     scheme: tls://       # optional, defaults to "" (which equals "dns://" in CoreDNS)
+#   - zone: foo.bar.
+#     scheme: udp://
```

Comment:
`udp://` as far as I can tell is [not a valid scheme](https://github.com/coredns/coredns/blob/master/corefile.5.md), I think you meant `dns://`

Reply:
Thanks, I indeed meant, dns://